From 905549ba488a7b29e6d060541a015c17a26ff6cf Mon Sep 17 00:00:00 2001
From: David King <dking@redhat.com>
Date: Thu, 17 Sep 2015 22:11:36 +0100
Subject: [PATCH 1/2] tests: Use check_PROGRAMS as the primary

The tests should only be built if required by make check.

https://bugs.freedesktop.org/show_bug.cgi?id=92036
---
 tests/Makefile.am.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Makefile.am.inc b/tests/Makefile.am.inc
index b493280..981d391 100644
--- a/tests/Makefile.am.inc
+++ b/tests/Makefile.am.inc
@@ -28,7 +28,7 @@ tests/services/xdg-app-session.service: session-helper/xdg-app-session.service.i

 test_doc_portal_DEPENDENCIES = tests/services/org.freedesktop.portal.Documents.service  tests/services/xdg-app-session.service

-noinst_PROGRAMS = $(TEST_PROGS)
+check_PROGRAMS = $(TEST_PROGS)

 TESTS=testdb test-doc-portal

--
2.5.2
